src/_guides/language/effective-dart/design.md

```diff
@@ -1003,24 +1035,27 @@ The language specifies that this check is done automatically and your `==`
 method is called only if the right-hand side is not `null`.
 
 {:.good-style}
+<?code-excerpt "misc/lib/effective_dart/design_good.dart (eq-dont-check-for-null)" replace="/operator ==|hashCode/[!$&!]/g"?>
 {% prettify dart %}
 class Person {
   final String name;
+  // ...
+  @override
+  [!operator ==!](other) => other is Person && name == other.name;
 
-  operator ==(other) =>
-      other is Person && name == other.name;
+  @override
+  int get [!hashCode!] => ...
```

Comment:
In the guideline just above we suggest "When you define ==, you also have to define hashCode", so it seems best to illustrate that in the "good" example.

Reply:
I'd also include the actual code: `=> name.hashCode`.
That said, there is an `// ...` above, so we could just say that the `hashCode` declaration is implied.

Reply:
I prefer your suggestion to include `=> name.hashCode`.